runtime/calamari/src/lib.rs

```diff
@@ -349,14 +349,14 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    /// Relay Chain `TransactionByteFee` / 10
-    pub const TransactionByteFee: Balance = mKMA / 100;
```

Comment:
I'm not sure like renaming will keep previous storage.

Reply:
This is how I create the key for these ---> `let key = hex::encode(Twox128::hash(b":TransactionByteFee:"));`
And when I check the value for this key is always `0x` , also tried with all lower case and upper case as well.